README.md

```diff
@@ -19,40 +19,111 @@
 Windows is not an explicitly supported environment, although where possible Palisade has been made compatible.  
 For Windows developer environments, we recommend setting up [WSL](https://docs.microsoft.com/en-us/windows/wsl/).
 
-# Palisade Examples
+## Overview of the Example
```

Comment:
I think this is too long and detailed, for this section I would suggest that we have something that is more like a scene setter about the example being set around a generic company needing to protect its employees HR data while maintaining wide legitimate access to that data. Therefore we have a number of users defined (point to user lib for details), a number of purposes defined (point to where they are explained in detail) and a set of policies (point to those) which act on the HR data (point to schema).